enhancements/proposals/2549-scylladbcluster-backup-repair/README.md

```diff
@@ -0,0 +1,438 @@
+# ScyllaDBCluster integration with ScyllaDB Manager: cluster registration, backup and repair scheduling
+
+## Summary
+
+This proposal aims at introducing initial support for ScyllaDBCluster integration with ScyllaDB Manager.
+The initial integration covers support for registration of managed, multi-datacenter clusters, and scheduling backup and repair tasks.
+
+## Motivation
+
+We want to introduce ScyllaDB Manager integration with ScyllaDBCluster to extend our existing integration to automated multi-datacenter clusters.
+We want to support all the operations we currently support with single-datacenter clusters to at least the same extent.
+This also allows us to deal with some of the issues and inconsistencies existing in ScyllaCluster's API related to ScyllaDB Manager integration.
+
+### Goals
+
+- Provide a suggested deployment mechanism for unmanaged ScyllaDB Manager server deployment in multi-datacenter Kubernetes environments.
+- Introduce support for registration of ScyllaDBClusters with ScyllaDB Manager.
+- Introduce support for scheduling of backup tasks for ScyllaDBClusters with ScyllaDB Manager.
+- Introduce support for scheduling of repair tasks for ScyllaDBClusters with ScyllaDB Manager.
+- Maintain support for registering ScyllaClusters and scheduling backup and repair tasks for ScyllaClusters with ScyllaDB Manager.
```

Comment:
Product-wise we're not "introducing support" but extending support to multi-DC clusters.
While this sentence is technically correct (because _technically_) we're introducing support to Scylla**DB**Clusters that was present in ScyllaClusters, this phrasing misses the practical sense, especially for outside readers.

Suggested (hopefully more accurate/useful) phrasing could be:

```suggestion
- Rework the CRD API for backup & repair tasks to decouple from the legacy `ScyllaCluster` CRD.
    - Includes a compatibility solution for legacy `ScyllaCluster` setups using the in-CRD backup/repair stanzas.
- Introduce a way to federate backup&repair tasks across Kubernetes clusters.
```

If you think it doesn't capture the meaning, feel free to correct me!